packages/style-loader/index.js

```diff
@@ -2,64 +2,103 @@ const ExtractTextPlugin = require('extract-text-webpack-plugin');
 const merge = require('deepmerge');
 
 module.exports = (neutrino, opts = {}) => {
+  const cssTest = neutrino.regexFromExtensions(['css']);
+  const cssModulesTest = neutrino.regexFromExtensions(['module.css']);
+
   const options = merge({
+    test: (input) => {
+      const isCssModule = cssModulesTest.test(input);
+      const isRegularCss = cssTest.test(input);
+
+      if (opts.modules !== false && isCssModule) {
```

Comment:
Since `opt.modules` is always just a bool, why not just do `opts.modules && isCssModule`?

Reply:
Hmm, for some reason I had it in my head that it could be an object, but I guess that's not the case.

Reply:
Ah, I know why: `opts.modules` could be undefined, and in that case, it should be treated the same as `true`, since that would be the default value.

Reply:
Ah gotcha.